app/assets/stylesheets/patternfly_and_overrides.scss

```diff
@@ -3,72 +3,78 @@
 
 $navbar-default-bg: $primary_color;
 
-@import "bootstrap";
+@import "patternfly";
 
-.dropdown-menu {
-  border-radius: 0 0 6px 6px;
+.select2-container .select2-choice {
+  height: 26px;
 }
 
-// Necessary because bootstrap3 nav dropdown does not have top arrow
-.nav li .dropdown-menu:after {
-  content: '';
-  display: inline-block;
-  border-left: 6px solid transparent;
-  border-right: 6px solid transparent;
-  border-bottom: 6px solid #ffffff;
-  position: absolute;
-  top: -6px;
-  left: 10px;
+.select2-arrow > b{
+
+    margin: -4px 0 0 0;
+
 }
 
-.navbar .navbar-right > li > .dropdown-menu:after,
-.navbar .nav > li > .dropdown-menu.pull-right:after {
-  left: auto;
-  right: 13px;
+#main {
+  padding: 0 40px;
 }
 
-.nav li.dropdown-submenu .dropdown-menu:after{
-  display: none;
+body {
+  font-size: 14px;
```

Comment:
The default recommended font-size by Patternfly is 12px, @ohadlevy @alongoldboim , [see their test pages](https://rawgit.com/patternfly/patternfly/master/tests/basic.html), the idea is to use patterns that have been user-tested, provide a common UI framework for enterprise apps and not deviate much from it as we're not going to do any testing. A simple example that comes to mind, [foreman_cockpit components](https://camo.githubusercontent.com/dd327f958993d60ecefbe4e1fc26bc45ad63cf9c/687474703a2f2f692e696d6775722e636f6d2f527a64735239622e676966) would not fit as well if we don't follow Patternfly recommendations.